core/src/main/java/jenkins/ClassLoaderReflectionToolkit.java

```diff
@@ -115,7 +201,7 @@ public Class findLoadedClass(ClassLoader cl, String name) throws InvocationTarge
     @Deprecated
     public Class findClass(ClassLoader cl, String name) throws InvocationTargetException {
         try {
-            return (Class)FIND_CLASS.invoke(cl,name);
+            return (Class)FindClass.FIND_CLASS.invoke(cl,name);
         } catch (IllegalAccessException e) {
```

Comment:
Wonder why these are not using `invoke`.

Reply:
hmm isn't it using invoke? or do you mean something else?

Reply:
I meant `ClassLoaderReflectionToolkit.invoke` rather than `Method.invoke`.